src/test/ui/intrinsics/unchecked_ctfe.rs

```diff
@@ -0,0 +1,25 @@
+#![feature(core_intrinsics)]
+#![feature(const_int_unchecked_arith)]
+
+const _: u16 = unsafe { std::intrinsics::unchecked_add(40000u16, 30000) };
+//~^ ERROR any use of this value will cause an error
+
+const _: u32 = unsafe { std::intrinsics::unchecked_sub(14u32, 22) };
+//~^ ERROR any use of this value will cause an error
+
+const _: u16 = unsafe { std::intrinsics::unchecked_mul(300u16, 250u16) };
+//~^ ERROR any use of this value will cause an error
+
+const _: i32 = unsafe { std::intrinsics::unchecked_div(1, 0) };
+//~^ ERROR any use of this value will cause an error
+
+const _: i32 = unsafe { std::intrinsics::unchecked_div(i32::min_value(), -1) };
+//~^ ERROR any use of this value will cause an error
+
+const _: i32 = unsafe { std::intrinsics::unchecked_rem(1, 0) };
+//~^ ERROR any use of this value will cause an error
+
+const _: i32 = unsafe { std::intrinsics::unchecked_rem(i32::min_value(), -1) };
+//~^ ERROR any use of this value will cause an error
+
+fn main() {}
```

Comment:
This covers most of them; what about `unchecked_shl` and `unchecked_shr`?

Reply:
Those weren't added in #68809 and are tested elsewhere.

Reply:
Fair, but wouldn't it make more sense to have them all together though?

Reply:
I don't care. Want me to move them?

Reply:
Yes, please do -- probably best to just add your new tests to that existing file.